src/client/backends/Msfs/Msfs.ts

```diff
@@ -187,34 +187,34 @@ export class MsfsBackend implements DataInterface {
     }
 
     /** @inheritdoc */
-    public async getWaypoints(idents: string[], ppos?: Coordinates, icaoCode?: string, airportIdent?: string): Promise<Waypoint[]> {
+    public async getWaypoints(idents: string[], ppos?: Coordinates, icaoCode?: string, airports?: string[]): Promise<Waypoint[]> {
         const results = new Map<string, Waypoint>();
 
-        for (const ident in idents) {
-            (await this.cache.searchByIdent(ident, IcaoSearchFilter.Intersections, 100)).forEach((v) => results.set(v.icao, this.mapping.mapFacilityToWaypoint(v)));
-        }
+        const promises = idents.map((ident) => this.cache.searchByIdent(ident, IcaoSearchFilter.Intersections, 100));
+
+        (await Promise.all(promises)).flat().forEach((v) => results.set(v.icao, this.mapping.mapFacilityToWaypoint(v)));
```

Comment:
This change achieves nothing other than making the code even less readable with the old-school array methods. If the lint rules are counter-productive we should fix them instead. Same applies for the other two similar changes in this file.